From 0370cf8a24e90382dcb7164188df95e263efed99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Mon, 5 Sep 2022 23:23:25 +0100
Subject: [PATCH] disable multilibs builds on el8, 9 and 9-next

---
 rfpkg/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rfpkg/__init__.py b/rfpkg/__init__.py
index ada3868..75b4e9f 100644
--- a/rfpkg/__init__.py
+++ b/rfpkg/__init__.py
@@ -190,8 +190,8 @@ class Commands(pyrpkg.Commands):
             'opencore-amr', 'rtmpdump', 'vo-amrwbenc', 'x264', 'x265', 'xvidcore', 'zsnes',
             'Cg', 'dega-sdl', 'gens', 'pcsx2', 'steam', 'xorg-x11-drv-nvidia',
             'xorg-x11-drv-nvidia-470xx', 'xorg-x11-drv-nvidia-390xx',
-            'xorg-x11-drv-nvidia-340xx', 'unace'] and self.branch_merge not in ['el6', 'f28',
-            'f29', 'f30'] and self.namespace in ['free', 'nonfree']:
+            'xorg-x11-drv-nvidia-340xx', 'unace'] and self.branch_merge not in ['el8', 'el9',
+            'el9-next'] and self.namespace in ['free', 'nonfree']:
             self._target += "-multilibs"
 
     def _tag2version(self, dest_tag):
-- 
2.37.2

