From 11f2c566d4513eac1303a4a38eff3418b442739e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Fri, 11 Feb 2022 23:20:56 +0000
Subject: [PATCH 2/2] Add unace to multilibs because is an i686 package

---
 rfpkg/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rfpkg/__init__.py b/rfpkg/__init__.py
index 30ac68c..d376d44 100644
--- a/rfpkg/__init__.py
+++ b/rfpkg/__init__.py
@@ -190,7 +190,7 @@ class Commands(pyrpkg.Commands):
             'opencore-amr', 'rtmpdump', 'vo-amrwbenc', 'x264', 'x265', 'xvidcore', 'zsnes',
             'Cg', 'dega-sdl', 'gens', 'pcsx2', 'steam', 'xorg-x11-drv-nvidia',
             'xorg-x11-drv-nvidia-470xx', 'xorg-x11-drv-nvidia-390xx',
-            'xorg-x11-drv-nvidia-340xx'] and self.branch_merge not in ['el6', 'f28',
+            'xorg-x11-drv-nvidia-340xx', 'unace'] and self.branch_merge not in ['el6', 'f28',
             'f29', 'f30'] and self.namespace in ['free', 'nonfree']:
             self._target += "-multilibs"
 
-- 
2.34.1

