From be33cce92923328b613e4245bfb28f2c10c93f35 Mon Sep 17 00:00:00 2001
From: Leigh Scott <leigh123linux@gmail.com>
Date: Sat, 25 Apr 2020 13:12:12 +0100
Subject: [PATCH] Add zsnes, gens and dega-sdl to multilibs

---
 rfpkg/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rfpkg/__init__.py b/rfpkg/__init__.py
index d364998..8517f72 100644
--- a/rfpkg/__init__.py
+++ b/rfpkg/__init__.py
@@ -223,8 +223,8 @@ def load_target(self):
         if self._package_name_spec in ['buildsys-build-rpmfusion', 'gpac', 'gstreamer1-libav',
             'gstreamer1-plugins-bad-freeworld', 'gstreamer1-plugins-ugly', 'faad2', 'ffmpeg',
             'libde265', 'libdca', 'libmms', 'libquicktime', 'libva-intel-driver', 'mjpegtools',
-            'opencore-amr', 'rtmpdump', 'vo-amrwbenc', 'x264', 'x265', 'xvidcore',
-            'Cg', 'pcsx2', 'steam', 'xorg-x11-drv-nvidia', 'xorg-x11-drv-nvidia-390xx',
+            'opencore-amr', 'rtmpdump', 'vo-amrwbenc', 'x264', 'x265', 'xvidcore', 'zsnes',
+            'Cg', 'dega-sdl', 'gens', 'pcsx2', 'steam', 'xorg-x11-drv-nvidia', 'xorg-x11-drv-nvidia-390xx',
             'xorg-x11-drv-nvidia-340xx'] and self.branch_merge not in ['el6', 'f28',
             'f29', 'f30'] and self.namespace in ['free', 'nonfree']:
             self._target += "-multilibs"
