--- a/src/rfpkg/__init__.py
+++ b/src/rfpkg/__init__.py
@@ -228,6 +228,10 @@
                            'default method: %s' % e)
             super(Commands, self).load_user()
 
+    def _tag2version(self, dest_tag):
+        """ get the '26' part of 'f26-foo' string """
+        return dest_tag.split('-')[0].replace('f', '')
+
     # New functionality
     def _findmasterbranch(self):
         """Find the right "rpmfusion" for master"""
@@ -235,11 +239,7 @@
         # If we already have a koji session, just get data from the source
         if self._kojisession:
             rawhidetarget = self.kojisession.getBuildTarget('rawhide-free')
-            desttag = rawhidetarget['dest_tag_name']
-            desttag=desttag.split('-')
-            desttag.remove('free')
-            desttag=''.join(desttag)
-            return desttag.replace('f', '')
+            return self._tag2version(rawhidetarget['dest_tag_name'])
         else:
             # We may not have Fedoras.  Find out what rawhide target does.
             try:
@@ -249,11 +249,7 @@
                 # We couldn't hit koji, bail.
                 raise pyrpkg.rpkgError('Unable to query koji to find rawhide \
                                        target')
-            desttag = rawhidetarget['dest_tag_name']
-            desttag=desttag.split('-')
-            desttag.remove('free')
-            desttag=''.join(desttag)
-            return desttag.replace('f', '')
+            return self._tag2version(rawhidetarget['dest_tag_name'])
 
     def _determine_runtime_env(self):
         """Need to know what the runtime env is, so we can unset anything

